app/views/fact_values/welcome.html.erb

```diff
@@ -1,12 +1,24 @@
 <% content_for(:title, _("Fact Values")) %>
+
 <div class="blank-slate-pf">
   <div class="blank-slate-pf-icon">
     <%= icon_text("list-alt", "", :kind => "fa") %>
   </div>
   <h1><%= _('Fact Values') %></h1>
-  <%= _("You don't seem to have any facts yet. If you wish to configure fact pushing, please follow the documentation.") %>
-  <div class="blank-slate-pf-main-action">
-    <%= link_to _('Documentation'), documentation_url("3.5.5FactsandtheENC"), :rel => "external",
-      :class => "btn btn-primary btn-lg" %>
-  </div>
+  <% if Setting[:enable_facts_storage] %>
+    <%= _("You don't seem to have any facts yet. If you wish to configure fact pushing, please follow the documentation.") %>
+    <div class="blank-slate-pf-main-action">
+      <%= link_to _('Documentation'), documentation_url("3.5.5FactsandtheENC"), :rel => "external",
```

Comment:
AFAIK fact storage is also used by subscription-manager in Katello. There's also the Ansible plugin which can upload facts.

Reply:
It's original content, not my change (same in `config_reports/welcome`)